gpt_computer_agent/__init__.py

```
@@ -2,6 +1,288 @@
+
+
+    def start(self):
+        command = "python -c 'from gpt_computer_agent import start; start(True);'"
+        self.process = subprocess.Popen(command, shell=True)
+
+
+    def close(self):
+        try:
+            self.client.stop_server()
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)